.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -27,6 +27,12 @@ fun shouldBeSkipped(subProject: GradleSubproject, testConfig: TestCoverage): Boo
     return testConfig.os.ignoredSubprojects.contains(subProject.name)
 }
 
+val cleanDaemonsBeforePerformanceTest = """
+    free -m
+    ps aux | grep GradleDaemon | grep -v grep | awk '{print ${'$'}2}' | xargs kill -9
```

Comment:
* you can filter out the grep command by using a regex instead: ```ps aux | grep '[Gg]radleDaemon'```
* you can probably escape the dollar sign by `awk { print \$2 }`

Reply:
> ps aux | grep '[Gg]radleDaemon'

Good to know this.

> you can probably escape the dollar sign by awk { print \$2 }

Thanks! But anyway we need to pass a string `{print $2}` to awk, so `awk '{print $2}'` is probably more readable than `awk {print\ \$2}`

Reply:
Minor: Just thinking out loud - would putting the whole command (and not just the dollar) in a separate string make it a bit more readable, i.e.
```kotlin
"""
    free -m
    ps aux | grep GradleDaemon | grep -v grep | ${"awk '{print \$2}'"} | xargs kill -9
"""
```
or
```kotlin
"""
    free -m
    ${"ps aux | grep GradleDaemon | grep -v grep | awk '{print \$2}' | xargs kill -9"}
"""
```